src/gui/GUI.java

```diff
@@ -49,6 +49,8 @@
     
     private ComputerWindowController computerWindowController;
     
+    private HelpWindowController helpWindowController;
```

Comment:
doc block. 😄

Reply:
fixed-in: 8db074ab0e284833c3be8adca8938e345183856a